src/tpm2_pytss/ESAPI.py

```diff
@@ -1687,7 +1687,7 @@ def ecdh_key_gen(
             TSS2_Exception: Any of the various TSS2_RC's the lower layers can return.
 
         Returns:
-            A Tuple[TPM2B_ECC_POINT, TPM2B_ECC_POINT] which is the zPoint Results of P := h[de]Qs
+            :A Tuple[TPM2B_ECC_POINT, TPM2B_ECC_POINT] which is the zPoint Results of P := h[de]Qs
```

Comment:
Why does this need a starting `:`?

Reply:
For some reason the documentation losses part of the line without, IIRC I tried a bunch of things but couldn't find anything better